File: Vanta-Scripts/Source/Input/Input.cs
using System;

namespace Vanta {

    public static class Input {
        public static bool IsKeyDown(KeyCode key) {
            return Internal.Input_IsKeyDown(key);
        }

        public static bool IsMouseDown(MouseButton button) {
            return Internal.Input_IsMouseDown(button);
        }
    }
}

File: Vanta/src/Vanta/Scripts/Native/Module/Input/Input.cpp
#include <vantapch.hpp>
#include "Vanta/Scripts/Native/Module/Input/Input.hpp"
#include "Vanta/Input/Input.hpp"

namespace Vanta {
    namespace NativeImpl {

        bool Input_IsKeyDown(KeyCode key) {
            return Input::IsKeyPressed(key);
        }

        bool Input_IsMouseDown(MouseCode button) {
            return Input::IsMouseButtonPressed(button);
        }
    }
}
